manuscript.md

```diff
@@ -289,7 +296,7 @@ explicit models of network structure.
 networks. Starting with the input of data from multiple sources, followed by a
 modelling framework for ecological networks and the landscape, which are then
 ultimately combined to allow for the prediction of spatially explicit
-networks.](figures/conceptual_v3.png){#fig:conceptual}
+networks.](figures/concept_v6.png){#fig:conceptual}
```

Comment:
I thought (but one I'm inclined to ignore) what do you think of the idea of incorporating validation in here as an additional step? I feel it deviates from the core idea of the figure but at the same time it is part of the whole process so it could work. Needn't be too comprehensive - maybe even adding the arc figure into here???

Reply:
in my opinion it would be easier to add the AUC concept figure to figure 3 (the poorly named `figures/forecasting_v3.png) as there is more room there and the concept of empirical features/labels is already clearly defined there and not as much in the fig 2

Reply:
Yeah I could get onboard with that... Makes the current AUC figure less 'random' but we still keep the info. Flexible either way though (keep separate or merge in here)